Adding Optional Second button and function

CaptainJeff opened the pull request:
This fork lets you add an optional button next to the "Next" button. This lets you call a function on a specific button which lets the tour be more interactive

booleanbetrayal commented:
Hi @CaptainJeff - This sounds like a potentially good feature, but I'll need a couple of things from you before I can look at it further. Looks like you modified one of the dynamically built amalgamations rather than the upstream source (`/src/tour/tour.js`). You can edit that file and then run `npm install && grunt` to run tests and rebuild the dist files. Also, it would be nice if you could rebase your commits, squashing them into a single commit.

Thanks and let me know if you have any questions.

CaptainJeff commented:
Will do man, didn't realize I should be changing that /src/tour/tour.js file. I also changed quite a bit. I added a few optional features. Some would call functions, others would just be a href, and there were some styling features. I'll re-do this and submit again. It'll probably take me a few days or a week.

booleanbetrayal commented:
Sounds good! If you have multiple independent features you'd like to get in, it'd make more sense to do individual PRs. You can always keep a private fork and cherry-pick individual commits as needed to support your project in the meantime.

booleanbetrayal commented:
Hey @CaptainJeff - Just seeing if you would like to clean up this PR (or file a new one). Thanks!
